controllers/placementapi_controller.go

```diff
@@ -1265,6 +1265,7 @@ func (r *PlacementAPIReconciler) generateServiceConfigMaps(
 		httpdVhostConfig[endpt.String()] = endptConfig
 	}
 	templateParameters["VHosts"] = httpdVhostConfig
+	templateParameters["TimeOut"] = instance.Spec.APITimeout
```

Comment:
I understand this will be eventually be read from `config/crd/bases/placement.openstack.org_placementapis.yaml`, where the default timeout is set.

any reason, we need to maintain same spec at `api/bases/placement.openstack.org_placementapis.yaml`

Reply:
@gibizer do you know perchance?

Reply:
timeout will be set from openstack operator lvl

Reply:
@auniyal61 the commit introduced the coping explains the reason https://github.com/openstack-k8s-operators/nova-operator/pull/450